web/src/components/tasks/LicenseTask.test.tsx

```diff
@@ -215,11 +218,11 @@ describe("<LicenseTask />", () => {
 
     it("does not display error alert when license status is found", async () => {
       renderTask();
-
       mockApi.checkLicenseStatus.mockResolvedValue(generateUserData({}));
```

Comment:
why is this test changing from Pending to Completed? Is seems odd. Either something is wrong or the staged data needs to be explicitly stated in this test.